### DISCUSSION TITLE
只能接收消息却没有任何反应不能回复咋办呀

### DISCUSSION THREAD

**Discussion by Darkerami** [nonebot/discussions#306] [问答]
![a85ce03f80e7eae7cd137ac94d2e5f7](https://user-images.githubusercontent.com/118607168/215322178-823dc16e-78ef-4923-a903-cde8fa2ee6a9.png)
连内置自带的/echo和/say都不能回复

**Comment by yanyongyu** [comment]
你根本没配上报地址

**Reply by Darkerami** [reply]
您好能再解释一下吗？是那个127.0.0.1:8080的地方吗？您说的上报地址？我需要换成什么？
Thank u!!!

**Comment by yanyongyu** [comment]
https://docs.go-cqhttp.org/guide/config.html#%E9%85%8D%E7%BD%AE%E4%BF%A1%E6%81%AF
onebot.adapters.nonebot.dev
请配置gocq的ws-reverse上报地址为`ws://127.0.0.1:你的nonebot端口号/onebot/v11/`

**Reply by Darkerami** [reply]
您说的这个端口号是查什么地方？

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215334146-66ee42e9-1f53-4222-8975-b0001c8d09fc.png)
是改这里吗？

**Reply by Darkerami** [reply]
这个本机nonebot的端口号是在什么地方能查到呢？

**Reply by Darkerami** [reply]
还是随意写呢？

**Reply by Darkerami** [reply]
您好我改了一下这里还是没有反应呀
![image](https://user-images.githubusercontent.com/118607168/215334731-21b88756-bfde-4e23-a71c-2f123ea72824.png)

**Reply by yanyongyu** [reply]
你nonebot启动后显示的端口号就是

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215336163-6e221ca4-3283-46ab-86ed-cb2b70c88447.png)
是这里的8080吗？我配置里写了呀

**Reply by Darkerami** [reply]
改完您说的那里后这个显示连接不上啊
![image](https://user-images.githubusercontent.com/118607168/215336213-76256c2d-231e-4bba-8c0a-76692e0b3431.png)

**Reply by yanyongyu** [reply]
？你nonebot启动了吗？

**Reply by Darkerami** [reply]
？终端运行和那个exe

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215336350-c049bb9d-0ef7-45e9-8ccb-c45a57d5730e.png)

**Reply by yanyongyu** [reply]
你运行的这个叫gocqhttp，不是你的nonebot

**Reply by yanyongyu** [reply]
nb1的上报地址是/ws

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215336419-03071517-624b-4819-9de4-fe86123755b1.png)

**Reply by Darkerami** [reply]
这个不是运行的吗？

**Reply by yanyongyu** [reply]
把ws路径改成`ws://127.0.0.1:8080/ws`

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215336557-7217c0ae-1b55-472c-98a6-6464fd31f63a.png)

**Reply by Darkerami** [reply]
这里改了呀

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215336641-3acd5fed-16fc-45d5-ae71-07f5fe19d100.png)

**Reply by yanyongyu** [reply]
你看看清楚我给你发的路径

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215337073-1680f76a-a814-4c11-9e5d-0438e6ea5e87.png)

**Reply by Darkerami** [reply]
是这里改成您说的吗？

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215337447-f07d5978-c217-49c6-81da-99e0899c0c7f.png)

**Reply by Darkerami** [reply]
您好我改了一下-ws  但是/echo这种命令还是执行不了诶：

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215337705-1ff9d785-3ba3-404f-84f5-6afd4fed9917.png)

**Reply by yanyongyu** [reply]
> 把ws路径改成`ws://127.0.0.1:8080/ws`

是把 ws-reverse的路径改成这个。。。另外截图请把nonebot和gocq的一起截你发一个没用

**Reply by Darkerami** [reply]
虽然问题没有解决依然感谢您这么晚还在帮助我
如果您能进一步指导将不胜感谢

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215338037-28b7298e-041d-4b70-a09d-46e7d1d80878.png)

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215338172-1614633f-f9fa-484b-b5fa-6327cc1c610b.png)

**Reply by yanyongyu** [reply]
把你gocq里的ws配置删掉，只留一个ws-reverse

**Reply by Darkerami** [reply]
![image](https://user-images.githubusercontent.com/118607168/215338295-4d3a924d-2c41-4711-8918-31be663cd588.png)

**Reply by Darkerami** [reply]
哇噻成功了！！！

**Reply by Darkerami** [reply]
她给我回复消息了！！！！

**Reply by Darkerami** [reply]
顺便再问一句，那个端口号是固定的8080吗？还是自己随便用呢

**Reply by Darkerami** [reply]
太感谢了！！

**Reply by yanyongyu** [reply]
端口号gocq和nonebot两边保持一致

**Reply by Darkerami** [reply]
okok谢谢谢谢您早点休息吧
